### PR TITLE
Create wpt_report fetch url

### PR DESCRIPTION
**Pull request by lukebjerring**
## Description
Further steps towards https://github.com/web-platform-tests/wpt.fyi/issues/714

Azure's artifacts are a zip-file, but there are some APIs for fetching contents from the zip directly.
This PR adds the logic for building those fetching URLs.

### CODE REVIEW

File: api/checks/azure.go [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+}
+
+// GetReportURL produces a URL for fetching the report file, if applicable.
+func (b BuildArtifact) GetReportURL(organization string) (*url.URL, error) {
+	containerID := b.Resource.GetContainerID()
+	if containerID == "" {
+		return nil, errors.New("Failed to extract container ID")
+	}
+	result, err := url.Parse(fmt.Sprintf(
+		"https://dev.azure.com/%s/_apis/resources/Containers/%v",
```
Comment (Hexcles) [inline api/checks/azure.go]: Link to documentation about "file containers"

File: api/checks/azure_test.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
+package checks
```
Comment (Hexcles) [inline api/checks/azure_test.go]: You forgot the "small" tag. (And Travis should complain; if not, we have a problem.)

File: api/checks/azure.go (hunk 1 above)
Comment (lukebjerring) [reply api/checks/azure.go]: I'd love to. It's undocumented. I found it using the UI. Clicking the ... reveals `Copy to clipboard`

image

File: api/checks/azure_test.go (hunk 2 above)
Comment (lukebjerring) [reply api/checks/azure_test.go]: Done.

**Comment by lukebjerring** [comment]
This PR endpoint requires login, so has been superseded by #714

**Pull request closed without merging**